src/datachain/sql/types.py

```diff
@@ -432,11 +503,49 @@ def json(self):
     def datetime(self):
         return None
 
-    def uuid(self):
+    def binary(self):
         return None
 
+
+class DBDefaults:
+    def string(self):
+        return sa.text("''")
+
+    def boolean(self):
+        return sa.text("False")
+
+    def int(self):
+        return sa.text("0")
```

Comment:
Are these all the default values in sqlite now also? Does it mean there are no longer null values in sqlite?

Reply:
It depends on how we are creating table. If we will pass `NOT NULL` (`nullable=False`) and default value when table is created — there will be no nulls.

But for SQLite we don't do this — nothing is changed.

Reply:
This class is used only if we set not nullable columns on table creation. So I have added tests for this case and updated these fields to provide correct values for SQLite implementation.